pen/static/pen/js/script.js

```diff
@@ -0,0 +1,47 @@
+var start = false;
+
+let i = -1;
+
+function a() {
+    i=(i+1)%38;
+    j=i-1;
+    if (i==0) {
+        j=37;
+    }
+    document.getElementById(i).className = "highlight";
+    document.getElementById(j).className = "highlighted";
+}
+
+document.onclick= function(event) {
+
+   if (!start) {
+        setInterval(a, 2000);
+        start = true;
+   }
+
+    if (event===undefined) event= window.event;
+    var target= 'target' in event? event.target : event.srcElement;
+
+    if (i>=0 && i<35) {
+        var text = document.getElementById('my_console').value;
+        var update_text = text + document.getElementById(i).value;
+        document.getElementById('my_console').value = update_text;
+    }
+
+    if (i==35) {
```

Comment:
Po submicie strona jest odświeżana. W efekcie po zapisaniu wiadomości wszystko znika i uzytkownik zostaje z pustą planszą. I nie ma jak wyświetlić zapisanej notatki. Trzeba jeszcze dopisać logikę która będzie wyświetlała pustą zawartość pola tekstowego jeżeli użytkownik nie ma zapisanej żadnej notatki, albo pole tekstowe z zapisaną notatką, jeżeli użytkownik ma zapisaną notatkę (to w widoku memo).

Reply:
Dopisano logikę, która wyświetla pustą zawartość pola tekstowego, jeżeli użytkownik nie ma zapisanej żadnej notatki albo wyświetla pole tekstowe z zapisaną notatką, jeśli użytkownik ma zapisaną notatkę (w widoku memo).